lib/index.js

```diff
@@ -0,0 +1,57 @@
+'use strict';
+
```

Comment:
This is just a build of the library to use in yarn in a simple way. It's not a real change which going to be pushed.